History.md

```diff
@@ -1,3 +1,8 @@
+## 5.3.3 / 2021-07-
```

Comment:
Do we still have in our notes somewhere that people should edit changelogs? This isn't necessary anymore, can be removed.

Reply:
👍  Removed.